components/private/Snapshot/MDCSnapshotTestCase.m

```diff
@@ -63,4 +63,36 @@ - (void)snapshotVerifyView:(UIView *)view {
   FBSnapshotVerifyView(imageView, nil);
 }
 
+// TODO(https://github.com/material-components/material-components-ios/issues/5888)
+// Support multiple OS versions and devices for snapshots
+- (BOOL)isSupportedDevice {
+  if (NSProcessInfo.processInfo.operatingSystemVersion.majorVersion != 11 ||
+      NSProcessInfo.processInfo.operatingSystemVersion.minorVersion != 2 ||
+      NSProcessInfo.processInfo.operatingSystemVersion.patchVersion != 0) {
+    NSLog(@"Skipping this test. Snapshot tests currently only run on iOS 11.2.0");
```

Comment:
Tip: Nit: Update the message to something like, 

```
NSLog(@"Unsupported device. Snapshot tests currently only run on iOS 11.2.0");
```

Reply:
This is only a nit because technically the method is no longer skipping tests, but testing for snapshot testing support.

Reply:
done